### ISSUE TITLE
Validation loss is greater than 1?

### ISSUE THREAD

**Issue by K-M-Ibrahim-Khalilullah** [matterport/Mask_RCNN#1691]
@waleedka @moorage @Cpruce 
I trained MaskRCNN on my own dataset. During training, some values are look like this.

> Epoch 32/40
200/200 [==============================] - 132s 660ms/step - loss: 0.1315 - rpn_class_loss: 9.4215e-04 - rpn_bbox_loss: 0.0107 - mrcnn_class_loss: 0.0074 - mrcnn_bbox_loss: 0.0088 - mrcnn_mask_loss: 0.1037 - val_loss: 1.6259 - val_rpn_class_loss: 0.1048 - val_rpn_bbox_loss: 0.5142 - val_mrcnn_class_loss: 0.2798 - val_mrcnn_bbox_loss: 0.1932 - val_mrcnn_mask_loss: 0.5338
Epoch 33/40
200/200 [==============================] - 133s 663ms/step - loss: 0.1308 - rpn_class_loss: 0.0012 - rpn_bbox_loss: 0.0111 - mrcnn_class_loss: 0.0069 - mrcnn_bbox_loss: 0.0090 - mrcnn_mask_loss: 0.1026 - val_loss: 1.3137 - val_rpn_class_loss: 0.0856 - val_rpn_bbox_loss: 0.4055 - val_mrcnn_class_loss: 0.2032 - val_mrcnn_bbox_loss: 0.1625 - val_mrcnn_mask_loss: 0.4570
Epoch 34/40
200/200 [==============================] - 133s 664ms/step - loss: 0.1240 - rpn_class_loss: 7.8069e-04 - rpn_bbox_loss: 0.0093 - mrcnn_class_loss: 0.0079 - mrcnn_bbox_loss: 0.0071 - mrcnn_mask_loss: 0.0989 - val_loss: 1.4614 - val_rpn_class_loss: 0.0831 - val_rpn_bbox_loss: 0.3221 - val_mrcnn_class_loss: 0.2878 - val_mrcnn_bbox_loss: 0.2030 - val_mrcnn_mask_loss: 0.5654

Why val_loss is greater than 1?
Which value will be better? How can I reduce?
Any suggestion please.
Thanks

**Comment by kimile599** [comment]
I got the same issue, the total loss is closing to .0 and still decreaing but the val loss is not converging at all.
